### PR TITLE
added german titles for S22

### PR DESCRIPTION
**Pull request by shinrax2**
except e06&07

### CODE REVIEW

File: data/anomalies.json [hunk 1]
```diff
@@ -92,5 +92,35 @@
...
+	"S22E07DE": {
+		"title": "Nobody Got Cereal"
+	},
+	"S22E08DE": {
+		"title": "Buddha Box"
+	},
+	"S22E09DE": {
+		"title": "Unerfüllt"
+	}
+	,"S22E10DE": {
```
Comment (craue) [inline data/anomalies.json]: misplaced `,`

**Comment by craue** [comment]
Would you use the titles given on [Wikipedia](https://de.wikipedia.org/wiki/South_Park/Episodenliste#Staffel_22)?

**Comment by shinrax2** [comment]
@craue thanks for the heads up

**Comment by craue** [comment]
Thank you, @shinrax2.

**Pull request merged**